src/js/extensions/auto-link.js

```diff
@@ -205,7 +205,7 @@ LINK_REGEXP_TEXT =
 
         createAutoLink: function (textNodes, href) {
             href = Util.ensureUrlHasProtocol(href);
-            var anchor = Util.createLink(this.document, textNodes, href),
+            var anchor = Util.createLink(this.document, textNodes, href, this.getEditorOption('targetBlank')),
```

Comment:
The purpose of `Util.createLink` is to build an anchor tag around a set of textnodes, and it will actually extract the text nodes from the DOM, and insert them into the anchor that is returned (presumably to be inserted somewhere by the user).

`Core.createLink` is your generic use case, where you just build an anchor around the user's current selection.

Now that you bring it up, I'm thinking we should rename `Util.createLink` or move it out of `Util` altogether. We can do that in a follow-up to this.